src/main/java/com/minecolonies/colony/jobs/JobGuard.java

```diff
@@ -0,0 +1,73 @@
+package com.minecolonies.colony.jobs;
+
+import com.minecolonies.client.render.RenderBipedCitizen;
+import com.minecolonies.colony.CitizenData;
+import com.minecolonies.entity.ai.basic.AbstractAISkeleton;
+import com.minecolonies.entity.ai.citizen.guard.EntityAIMeleeGuard;
+import com.minecolonies.entity.ai.citizen.guard.EntityAIRangeGuard;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Random;
+
+/**
+ * Job class of the guard.
+ */
+public class JobGuard extends AbstractJob
+{
+    /**
+     * The higher the number the lower the chance to spawn a knight. Default: 3, 50% chance.
+     */
+    private static final int GUARD_CHANCE = 3;
+    private GuardJob task = GuardJob.RANGER;
+
+    /**
+     * Public constructor of the farmer job.
+     *
+     * @param entity the entity to assign to the job.
+     */
+    public JobGuard(CitizenData entity)
+    {
+        super(entity);
+    }
+
+    @NotNull
+    @Override
+    public String getName()
+    {
+        return "com.minecolonies.job.Guard";
+    }
+
+    @NotNull
+    @Override
+    public RenderBipedCitizen.Model getModel()
+    {
+        int chance = new Random().nextInt(GUARD_CHANCE);
+        if (chance == 1)
+        {
+            task = GuardJob.KNIGHT;
+            return RenderBipedCitizen.Model.KNIGHT_GUARD;
+        }
+        task = GuardJob.RANGER;
+        return RenderBipedCitizen.Model.ARCHER_GUARD;
+    }
+
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)